src/open_inwoner/openzaak/models.py

```diff
@@ -65,23 +65,26 @@ class ZGWApiGroupConfig(models.Model):
         on_delete=models.PROTECT,
```

Comment:
Do we still need this if the service is required? Same for the other services below?